packages/builder-util-runtime/src/httpExecutor.ts

```diff
@@ -12,7 +12,7 @@ import { ProgressCallbackTransform, ProgressInfo } from "./ProgressCallbackTrans
 const debug = _debug("electron-builder")
 
 export interface RequestHeaders extends OutgoingHttpHeaders {
-  [key: string]: string
+  [key: string]: string | string[] | number
```

Comment:
What about using?
```
[key: string]: OutgoingHttpHeader | undefined
```
It's a standard @node-type that would update as node versions update. It's a type alias for
```
    type OutgoingHttpHeader = number | string | string[];
```

Reply:
Also, after changing the value. Please run `pnpm generate-all` to update relevant documentation (if any is needed)

Reply:
Please pull in latest `master`, changes were made to `httpExecutor` in another PR